### ISSUE TITLE
E211 false positive - match statement

### ISSUE THREAD

**Issue by DolajoCZ** [PyCQA/pycodestyle#1020]
Hello, I would like to report false positive error E211 in new version of python 3.10.

```python
def test(value: tuple[int, int]) -> None:
    """Test."""
    match value:
        case (0, 1):  # E211 whitespace before '('
            print("0-1")
        case (1, 2):  # E211 whitespace before '('
            print("1-2")
        case _:
            print("Other")


if __name__ == "__main__":
    test((1, 2))
```

- python version: 3.10.0 - 64 bit
- pycodestyle version: 2.7.0

Thank you

**Comment by DolajoCZ** [comment]
Sorry, I did not notice that this error is already corrected on [this](https://github.com/PyCQA/pycodestyle/blob/master/pycodestyle.py#L844) line.
